_includes/checklist.md

```diff
@@ -1,41 +1,91 @@
-* **Main repository:** <url>
-* **Running libraries:**
-    * <url>
-    * ...
-* **Site:** <url>
-* **Product manager:** @<username>
-* **Technical point of contact:** @<username>
+* **Main repository:** [url]
+* **Site:** [url]
+* **Product manager:** @[username]
+* **System Owner:** @[username of technical point of contact who will be the project representative in the Sprint]
+* **Infrastructure Lead:** @[username]
+* **ATO folder:** [url]
+* **Sprint notes:** [url]
 
 ## TODOs
 
-### Project team
+### ATO Sprint prerequisites
```

Comment:
Can we add a descriptor here along the lines of "These three checklists (Infra Lead and Project Team Technical and Project Team Content) must be complete before you may enter a sprint"